services/manage/app/router.js

```diff
@@ -52,6 +52,7 @@ Router.map(function() {
       });
     });
     this.route('lineitems');
+    this.route('reporting');
```

Comment:
Can we follow the same route naming convention and call this `reports` instead of `reporting`?